tests/device_tests/test_msg_recoverydevice_slip39_basic_dryrun.py

```diff
@@ -35,7 +35,7 @@
 ]
 
 
-@pytest.mark.setup_client(mnemonic=SHARES_20_2of3[0:2], passphrase=True)
+@pytest.mark.setup_client(mnemonic=SHARES_20_2of3[0:2], passphrase="TREZOR")
```

Comment:
is passphrase even used here?

Reply:
It is not, neither in the second testcase in this file

Reply:
i don't see a reason why passphrase _should_ be enabled, so please remove them (and the second file as well)

Reply:
Fixed in 40530e8